### ISSUE TITLE
Bugs and Typo fixes

### ISSUE THREAD

**Issue by Coldestadam** [BreakoutMentors/Data-Science-and-Machine-Learning#23]
## Why
Fixing typos or bugs in the main branch

### Breakdown
Explain your approach to implementing the change. Include the changes you will be making in each repository.
- [x] Neural Network lesson is corrupted
  - [x] Also change the name of the model from Softmax model to Neural Network
- [x] Update challenges to use PyTorch
- [x] Colab button for CNN lesson is not working
- [x] Change url of dataset in Multiple Linear Regression lesson
  - [x] Update lr = 0.1
  - [x] Update Error Distribution
- [x] Fashion-MNIST Neural network challenge needs to be updated
- [x] CNN stride left is not 1 but 2

### QA Plan
Steps you will follow locally and on the cloud to verify expected behavior of all affected areas.
- [ ] ∞

### Post Release
Non destructive QA we can perform post release to verify things are working.
- [ ] ∞
